### DISCUSSION TITLE
How to replace "Doks" in top left of screen

### DISCUSSION THREAD

**Discussion by purplefox** [thuliteio/doks#905] [General]
Hi, I am new to Doks and interested in using it for my project website.

I would like to replace "Doks" in the top left of the screen (which links to the home page) with my project logo, but I couldn't figure out how to configure doks to do that. 

Any help greatly appreciated

**Comment by h-enk** [comment]
No setting for that — yet. This is where your logo would go: https://github.com/h-enk/doks/blob/master/layouts/partials/header/header.html#L15
